README.md

```diff
@@ -1,56 +1,37 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Hello-Microverse
+In this project I set up the linters and add a html and css file also .gitignore file.The main purpose of this project is to setting up the linters.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Major languages used html and css
+- Frameworks used visual studio code
+- Technologies used Laptop
```

Comment:
[OPTIONAL] kindly note that visual studio code is not a frame work but an [ide](https://en.wikipedia.org/wiki/Integrated_development_environment) 
also by major technologies, the template wants you to refer to the technology stacks used such as react or ruby which you will learn about later but you have not yet covered.

-I would recommend you customize the template according to the project. For instance, this project has no frameworks or technologies.